testing/btest/coverage/default-load-baseline.test

```diff
@@ -7,7 +7,7 @@
 # prefix to make the test work everywhere. That's what the sed magic
 # below does. Don't ask. :-)
 
-# @TEST-REQUIRES: test -x ${BUILD}/auxil/spicy/spicy/bin/spicy-config
+# @TEST-REQUIRES: have-spicy
```

Comment:
```suggestion
# @TEST-REQUIRES: ${SCRIPTS}/have-spicy
```
Or, does that happen to be in the path? Elsewhere it's used with the `${SCRIPTS}` prefix so minimally would be nice to be consistent.

I've found `TEST-REQUIRES` lines to be a mine-field, because command not found, or command-failure is hard to distinguish from a failure result. There might be a btest feature lingering :-)

Reply:
Yeah, it's in `PATH`, but doesn't hurt for consistency.

Reply:
> I've found `TEST-REQUIRES` lines to be a mine-field, because command not found, or command-failure is hard to distinguish from a failure result. There might be a btest feature lingering :-)

Yeah, indeed. I have been thinking about switching to using a particular exit code for "skip".